homework2.ts

```diff
@@ -0,0 +1,64 @@
+interface AreaInterface {
+    makeElement() : any;
+}
+
+class Square implements AreaInterface {
+    name: string;
+    width: number;
+  
+    constructor(name: string, width: number) {
+        this.name = name;
+        this.width = width;
+    }
+  
+    makeElement() {
+        let area: number;
+        area = this.width * this.width;
+        return "The area of " + this.name + " is " + area;
```

Comment:
Because we are calculating area,  and we can use it in other place, so that, you should return a number contain area value only.